README.md

```diff
@@ -1,21 +1,34 @@
 # Cognite Glimpse
 
-This repository is a showcase of how to build apps on top of [Cognite Data Fusion](https://cognite.com/cognite/cognite-data-fusion/developers/).This app is mainly useful for manufacturing operators, To perform the CNC machines as efficiently as possible and to understand when they are not performing at their best.
+This repository is a showcase of how to build dashboard style apps on top of [Cognite Data Fusion](https://cognite.com/cognite/cognite-data-fusion/developers/). It can be used to build configurable dashboards showing various type of information such as events, time series, meta fields, shift utilization, etc which can be used to give a high-level overview about efficiency of a manufacturing process for example.
+
+Information is presented using a set of widgets which the operator can configure. Each widget is laid out in a fixed sized grid. Position of widgets in the grid can be changed based on operator's preferences. Overall dashboard configuration is persisted for each user separately.
+
+## Overview
+
+This is a react application and used redux for managing the state. We are using Redux-saga for data fetching (asynchronous state management).
```

Comment:
Application is based on React/Redux. We use Redux-saga for data fetching (asynchronous state management)..